docs/pages/modules/supporting-platforms.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Supporting more platforms
+description: Learn how to add support for macOS and tvOS platforms.
+---
+
+import { Step } from '~/ui/components/Step';
+
+Expo Modules API provides the first-class support for Android and iOS, however since all Apple platforms are based on the same foundation and use the same programming language,
+it is possible to target other Apple platforms in the Expo module. At the moment only **macOS** and **tvOS** platforms (also called [Out-of-Tree platforms](https://reactnative.dev/docs/out-of-tree-platforms)) are supported.
+This guide will walk you through adding support for these platforms.
+
+<Step label="1">
+
+### Use the `"apple"` platform in `expo-module.config.json`
+
+In order to provide more seamless support for other Apple platforms, Expo SDK 50 introduced a universal `"apple"` platform to instruct the autolinking that the module may support any of the Apple platform
+and the question whether to link the module in the specific CocoaPods target is moved off to the podspec.
+If you have used `"ios"` before, you can safely replace it.
+
+```diff expo-module.config.json
+{
+-   "platforms": ["ios"],
+-   "ios": {
+-     "modules": ["MyModule"]
+-   }
++   "platforms": ["apple"],
++   "apple": {
++     "modules": ["MyModule"]
++   }
+}
+```
+
+</Step>
+<Step label="2">
+
+### Update the podspec to declare support for other platforms
+
+The module's podspec needs to be updated with the supported platforms, otherwise CocoaPods would fail to install the pod on targets for the other platforms.
```

Comment:
```suggestion
The module's podspec needs to be updated with a list of the supported platforms. Otherwise, CocoaPods would fail to install the pod on targets for the other platforms.
```

Let's break the sentence for better readability